README.md

```
@@ -71,15 +71,28 @@ databricks volumes create maven default packages MANAGED
 databricks fs cp dist/mlops_with_databricks-0.0.1-py3-none-any.whl dbfs:/Volumes/maven/default/packages
 
 # Overwrite Package
-databricks fs cp dist/credit_default_databricks-0.0.5-py3-none-any.whl dbfs:/Volumes/maven/default/packages --overwrite
+databricks fs cp dist/credit_default_databricks-0.0.7-py3-none-any.whl dbfs:/Volumes/maven/default/packages --overwrite
 ```
 
 ## Data
 
 Default of Credit Card Clients Dataset
 https://www.kaggle.com/datasets/uciml/default-of-credit-card-clients-dataset/data
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)